### ISSUE TITLE
Training and Validation loss decreases but validation accuracy remains constant. Please help!!

### ISSUE THREAD

**Issue by Ziona16** [Ziona16/fo#1]
Train on 948 samples, validate on 237 samples
Epoch 1/100
948/948 [==============================] - 64s 68ms/step - loss: 1.2481 - accuracy: 0.5686 - val_loss: 0.8204 - val_accuracy: 0.5738
Epoch 2/100
948/948 [==============================] - 65s 68ms/step - loss: 0.8148 - accuracy: 0.5738 - val_loss: 0.8048 - val_accuracy: 0.5738
Epoch 3/100
948/948 [==============================] - 62s 66ms/step - loss: 0.8028 - accuracy: 0.5738 - val_loss: 0.7911 - val_accuracy: 0.5738
Epoch 4/100
948/948 [==============================] - 63s 66ms/step - loss: 0.7884 - accuracy: 0.5738 - val_loss: 0.7791 - val_accuracy: 0.5738
Epoch 5/100
948/948 [==============================] - 64s 67ms/step - loss: 0.7751 - accuracy: 0.5738 - val_loss: 0.7672 - val_accuracy: 0.5738
Epoch 6/100
948/948 [==============================] - 66s 69ms/step - loss: 0.7665 - accuracy: 0.5738 - val_loss: 0.7577 - val_accuracy: 0.5738
Epoch 7/100
948/948 [==============================] - 65s 69ms/step - loss: 0.7552 - accuracy: 0.5738 - val_loss: 0.7494 - val_accuracy: 0.5738
Epoch 8/100
948/948 [==============================] - 64s 68ms/step - loss: 0.7471 - accuracy: 0.5738 - val_loss: 0.7417 - val_accuracy: 0.5738
Epoch 9/100
948/948 [==============================] - 64s 68ms/step - loss: 0.7406 - accuracy: 0.5738 - val_loss: 0.7367 - val_accuracy: 0.5738
Epoch 10/100
948/948 [==============================] - 65s 68ms/step - loss: 0.7332 - accuracy: 0.5738 - val_loss: 0.7300 - val_accuracy: 0.5738
Epoch 11/100
948/948 [==============================] - 65s 69ms/step - loss: 0.7274 - accuracy: 0.5738 - val_loss: 0.7232 - val_accuracy: 0.5738
Epoch 12/100
948/948 [==============================] - 66s 69ms/step - loss: 0.7229 - accuracy: 0.5738 - val_loss: 0.7187 - val_accuracy: 0.5738
Epoch 13/100
948/948 [==============================] - 65s 69ms/step - loss: 0.7191 - accuracy: 0.5738 - val_loss: 0.7158 - val_accuracy: 0.5738
Epoch 14/100
948/948 [==============================] - 65s 69ms/step - loss: 0.7134 - accuracy: 0.5738 - val_loss: 0.7149 - val_accuracy: 0.5738
Epoch 15/100
948/948 [==============================] - 65s 68ms/step - loss: 0.7100 - accuracy: 0.5738 - val_loss: 0.7085 - val_accuracy: 0.5738
Epoch 16/100
948/948 [==============================] - 65s 69ms/step - loss: 0.7083 - accuracy: 0.5738 - val_loss: 0.7051 - val_accuracy: 0.5738
Epoch 17/100
948/948 [==============================] - 64s 68ms/step - loss: 0.7044 - accuracy: 0.5738 - val_loss: 0.7022 - val_accuracy: 0.5738
Epoch 18/100
948/948 [==============================] - 65s 68ms/step - loss: 0.7011 - accuracy: 0.5738 - val_loss: 0.7008 - val_accuracy: 0.5738
Epoch 19/100
948/948 [==============================] - 64s 67ms/step - loss: 0.7014 - accuracy: 0.5738 - val_loss: 0.6989 - val_accuracy: 0.5738
Epoch 20/100
948/948 [==============================] - 65s 69ms/step - loss: 0.6965 - accuracy: 0.5738 - val_loss: 0.6985 - val_accuracy: 0.5738
Epoch 21/100
948/948 [==============================] - 65s 68ms/step - loss: 0.6974 - accuracy: 0.5738 - val_loss: 0.6945 - val_accuracy: 0.5738
Epoch 22/100
948/948 [==============================] - 64s 67ms/step - loss: 0.6961 - accuracy: 0.5738 - val_loss: 0.6930 - val_accuracy: 0.5738
Epoch 23/100
948/948 [==============================] - 68s 72ms/step - loss: 0.6929 - accuracy: 0.5738 - val_loss: 0.6918 - val_accuracy: 0.5738
Epoch 24/100
948/948 [==============================] - 65s 69ms/step - loss: 0.6916 - accuracy: 0.5738 - val_loss: 0.6919 - val_accuracy: 0.5738
Epoch 25/100
948/948 [==============================] - 65s 69ms/step - loss: 0.6911 - accuracy: 0.5738 - val_loss: 0.6897 - val_accuracy: 0.5738
Epoch 26/100
948/948 [==============================] - 64s 68ms/step - loss: 0.6905 - accuracy: 0.5738 - val_loss: 0.6896 - val_accuracy: 0.5738
Epoch 27/100
948/948 [==============================] - 63s 67ms/step - loss: 0.6898 - accuracy: 0.5738 - val_loss: 0.6880 - val_accuracy: 0.5738
Epoch 28/100
948/948 [==============================] - 63s 67ms/step - loss: 0.6893 - accuracy: 0.5738 - val_loss: 0.6882 - val_accuracy: 0.5738
Epoch 29/100
948/948 [==============================] - 64s 67ms/step - loss: 0.6886 - accuracy: 0.5738 - val_loss: 0.6873 - val_accuracy: 0.5738
Epoch 30/100
948/948 [==============================] - 64s 67ms/step - loss: 0.6875 - accuracy: 0.5738 - val_loss: 0.6863 - val_accuracy: 0.5738
Epoch 31/100
948/948 [==============================] - 64s 68ms/step - loss: 0.6880 - accuracy: 0.5738 - val_loss: 0.6859 - val_accuracy: 0.5738
Epoch 32/100
948/948 [==============================] - 64s 68ms/step - loss: 0.6872 - accuracy: 0.5738 - val_loss: 0.6862 - val_accuracy: 0.5738
Epoch 33/100
948/948 [==============================] - 64s 68ms/step - loss: 0.6858 - accuracy: 0.5738 - val_loss: 0.6862 - val_accuracy: 0.5738
Epoch 34/100
948/948 [==============================] - 65s 69ms/step - loss: 0.6863 - accuracy: 0.5738 - val_loss: 0.6854 - val_accuracy: 0.5738
Epoch 35/100
948/948 [==============================] - 63s 67ms/step - loss: 0.6859 - accuracy: 0.5738 - val_loss: 0.6849 - val_accuracy: 0.5738
Epoch 36/100
948/948 [==============================] - 64s 68ms/step - loss: 0.6852 - accuracy: 0.5738 - val_loss: 0.6846 - val_accuracy: 0.5738
Epoch 37/100
948/948 [==============================] - 64s 67ms/step - loss: 0.6845 - accuracy: 0.5738 - val_loss: 0.6842 - val_accuracy: 0.5738
Epoch 38/100
948/948 [==============================] - 63s 66ms/step - loss: 0.6843 - accuracy: 0.5738 - val_loss: 0.6839 - val_accuracy: 0.5738
Epoch 39/100
948/948 [==============================] - 65s 68ms/step - loss: 0.6842 - accuracy: 0.5738 - val_loss: 0.6837 - val_accuracy: 0.5738
Epoch 40/100
948/948 [==============================] - 68s 72ms/step - loss: 0.6837 - accuracy: 0.5738 - val_loss: 0.6835 - val_accuracy: 0.5738
Epoch 41/100
948/948 [==============================] - 65s 69ms/step - loss: 0.6836 - accuracy: 0.5738 - val_loss: 0.6834 - val_accuracy: 0.5738
Epoch 42/100
948/948 [==============================] - 66s 70ms/step - loss: 0.6831 - accuracy: 0.5738 - val_loss: 0.6832 - val_accuracy: 0.5738
Epoch 43/100
948/948 [==============================] - 66s 70ms/step - loss: 0.6832 - accuracy: 0.5738 - val_loss: 0.6831 - val_accuracy: 0.5738
